modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc

```diff
@@ -27,96 +31,183 @@ The repository must be accessible from the hub cluster and be defined as a sourc
 
 .Procedure
 
-. Create a `ConfigMap` resource that describes the NICs for a group of hosts. For example:
+. Create three `ConfigMap` CRs that contain the group and site configuration. 
```

Comment:
Suggestion - since the following substeps are all parts of doing the action in this step, I think a colon might indication a stronger relationship. Up to you though.
```suggestion
. Create three `ConfigMap` CRs that contain the group and site configuration:
```